fernet_fields/fields.py

```diff
@@ -100,6 +90,22 @@ def validators(self):
             del self.__dict__['_internal_type']
 
 
+def get_prep_lookup(self):
+    """Handles lookups for EncryptedField fields"""
+    if self.lookup_name == 'isnull':
+        return super(self.__class__, self).get_prep_lookup()
+    else:
+        raise FieldError("Encrypted fields do not support lookups.")
+
+
+# Register all field lookups to our handler
+for name, lookup in models.Field.class_lookups.items():
```

Comment:
@brad I think we can simplify things slightly by not registering a new lookup for `isnull`, then we can remove the `if`/`else` in `get_prep_lookup` and just raise the `FieldError` in all cases.